service_management/azure/test/integration/service_bus/queues_scenario_test.rb

```diff
@@ -13,18 +13,36 @@
 # limitations under the License.
 #--------------------------------------------------------------------------
 require 'integration/test_helper'
+require 'integration/service_bus/scenario_helper'
 
-describe 'ServiceBus Queues Scenario' do
-  let(:queue_name) { ServiceBusQueueNameHelper.name }
-
+describe Azure::ServiceBus::ServiceBusService do
   subject { Azure::ServiceBus::ServiceBusService.new }
-  after { ServiceBusQueueNameHelper.clean }
+  let(:queue_name) { 'test-queue' }
+
+  describe 'Service bus queue scenario' do
+    before do
+      VCR.insert_cassette "service_bus/#{name}"
+    end
+
+    after do
+      subject.delete_queue queue_name
+      VCR.eject_cassette
+    end
+
+    it 'should upload many messages and read them back' do
```

Comment:
My thinking was moving deletion in the test itself may lead to not deleting queue when test fails. In those cases it'd helpful for us to make sure we delete the thing in the after block with reason 'making sure that test doesn't leak resources in the failure case' [Now this only will come up when we will re-recording them in future]. 
Any views/input on that?